src/almanack/cli.py

```diff
@@ -0,0 +1,48 @@
+"""
+Setup almanack CLI through python-fire
+"""
+
+import json
+
+import fire
+
+from .metrics.data import get_table
+
+
+def get_table_json_wrapper(repo_path: str) -> str:
+    """
+    Converts the output of get_table to a proper JSON str.
+    Note: python-fire seems to convert lists of dictionaries
+    into individual dictionaries without distinction within list.
+
+    Args:
+        repo_path (str):
+            Path to the repository from which to retrieve and
+            convert the table data.
+
+    Returns:
+        str:
+            JSON string representation of the table data.
+    """
+
+    return json.dumps(get_table(repo_path=repo_path))
+
+
+def cli_get_table() -> None:
+    """
+    Creates Fire CLI for get_table_json_wrapper.
+
+    This enables the use of CLI such as:
+    `almanck <repo path>`
+    """
+    fire.Fire(get_table_json_wrapper)
+
+
+if __name__ == "__main__":
+    """
+    Setup the CLI with python-fire for the almanack package.
+
+    This allows the function `check` to be ran through the
+    command line interface.
+    """
+    cli_get_table()
```

Comment:
I looked into this a bit, and it seems if your method returns a list it outputs each element of that list with newlines in between, which IMO constitutes distinction between the dicts, but still, I agree that it's a little odd. You can override this behavior when executing `fire.Fire()` by including the `serialize` argument, e.g.:

```suggestion
import json

import fire

from .metrics.data import get_table

def cli_get_table() -> None:
    """
    Creates Fire CLI for get_table_json_wrapper.

    This enables the use of CLI such as:
    `almanack <repo path>`
    """
    fire.Fire(get_table, serialize=json.dumps)


if __name__ == "__main__":
    """
    Setup the CLI with python-fire for the almanack package.

    This allows the function `check` to be ran through the
    command line interface.
    """
    cli_get_table()
```


*(I'm not sure why the diff shows my change as replacing the entire file, but I just deleted the wrapper function and fixed a typo in "almanack" in the `cli_get_table` docstring.)*

Reply:
Thanks for this! I'll add this in as an additional change.